vendor-bin/phan/composer.json

```diff
@@ -1,10 +1,5 @@
 {
-  "config": {
-    "platform": {
-      "php": "7.1"
-    }
-  },
```

Comment:
This was committed in the original setup of the phan infrastructure a long time ago.
We don't want that any more. We want phan to use whatever PHP version is the default at run-time (so it will run appropriately in each CI pipeline).